Maven/tools/chocolateyInstall.ps1

```diff
@@ -16,8 +16,8 @@ $m2_repo = Join-Path $env:USERPROFILE '.m2'
 
 $url = "https://archive.apache.org/dist/maven/maven-3/$version/binaries/$name-bin.zip"
 
-
-[Environment]::SetEnvironmentVariable('M2_HOME', $m2_home, "User")
+# Delete leftovers from previous versions
+Remove-Item "$package\apache-maven-*" -Force -Recurse

```

Comment:
Use of backslash is not recommended. See [here](https://devblackops.io/joining-paths-in-powershell/). Many packages uses Join-Path instead.

Else it looks good to me.

Reply:
Good catch. Done!